dashed/assets/javascripts/welcome.js

```diff
@@ -0,0 +1,53 @@
+var $ = window.$ = require('jquery');
+var jQuery = window.jQuery = $;
+var px = require('./modules/dashed.js');
+
+require('../stylesheets/dashed.css');
+require('../stylesheets/welcome.css');
+
+require('bootstrap');
+require('datatables');
+require('d3');
```

Comment:
it seems to work after removing both dash.js (which I don't need) and d3... I'll keep it off then